AK/StackInfo.cpp

```diff
@@ -12,7 +12,7 @@
 
 #ifdef __serenity__
 #    include <serenity.h>
-#elif defined(__linux__) or defined(AK_OS_MACOS)
+#elif defined(__linux__) or defined(AK_OS_MACOS) or defined(__FreeBSD__)
```

Comment:
We have `AK_BSD_GENERIC`, it would be better to use that instead of just checking for FreeBSD here? 

https://github.com/SerenityOS/serenity/blob/master/AK/Platform.h#L27

Reply:
i wasn't aware that was available. will use that then, thanks!